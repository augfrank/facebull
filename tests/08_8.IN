M0   C0   C1  2066
M1   C0   C2  2213
M2   C0   C3  2662
M3   C0   C4  2479
M4   C0   C5  2466
M5   C0   C6  2741
M6   C1   C0  2473
M7   C1   C2  2146
M8   C1   C3  2026
M9   C1   C4  2776
M10   C1   C5  2883
M11   C1   C6  2695
M12   C1   C7  2021
M13   C2   C0  2033
M14   C2   C1  2605
M15   C2   C3  2454
M16   C2   C4  2336
M17   C2   C5  2296
M18   C2   C6  2743
M19   C3   C0  2430
M20   C3   C1  2863
M21   C3   C4  2547
M22   C3   C5  2779
M23   C3   C6  2618
M24   C3   C7  2794
M25   C4   C0  2400
M26   C4   C1  2466
M27   C4   C2  2061
M28   C4   C3  2548
M29   C4   C5  2654
M30   C4   C6  2386
M31   C4   C7  2299
M32   C5   C0  2468
M33   C5   C1  2861
M34   C5   C3  2694
M35   C5   C4  2919
M36   C5   C7  2504
M37   C6   C0  2417
M38   C6   C1  2743
M39   C6   C2  2845
M40   C6   C3  2795
M41   C6   C4  2006
M42   C6   C5  2941
M43   C6   C7  2032
M44   C7   C1  2734
M45   C7   C2  2973
M46   C7   C3  2595
M47   C7   C4  2921
M48   C7   C5  2077
M49   C7   C6  2253
