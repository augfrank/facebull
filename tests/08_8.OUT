17576
0 12 13 23 27 34 41 48
